comment/comment.go

```diff
@@ -89,6 +90,24 @@ func (m *GormCommentRepository) Save(ctx context.Context, comment *Comment) (*Co
 	return comment, nil
 }
 
+// Delete a single comment
+func (m *GormCommentRepository) Delete(ctx context.Context, comment *Comment) error {
+	c := Comment{}
+	tx := m.db.Where("id=?", comment.ID).First(&c)
+	if tx.RecordNotFound() {
```

Comment:
@Ritsyy but @aslakknutsen is right in that you should delete a comment based on it's UUID and not based on the full comment object.  

Please, make it

```go
Delete(ctx context.Context, id uuid.UUID) error
```